docs/JLBP-0018.md

```diff
@@ -22,9 +22,10 @@ There are a number of problems with shading:
   to also roll in the security fix).
 - Shading cannot be performed on types used in a library's own public API as return
   types or method arguments.
-- Service entries under `META-INF/services` can easily be messed up. They need
-  special effort to be merged because the entries are located in the same place
-  for every jar.
+- Shading often corrupts non-code metadata in META-INF/MANIFEST.mf and other locations.
+  For instance, service entries under `META-INF/services` are hard 
+  to merge because the entries are located in the same place
+  for every JAR file.
```

Comment:
Yes, that's one example of the sorts of problems that can happen. Not every shading will have this problem, or indeed any problem with metadata, but it can crop up.